external/slsi_wifi/slsi_wifi_api.c

```diff
@@ -1485,8 +1487,8 @@ void slsi_monitor_thread_handler(void *param)
 		}
 	}
 	VPRINT("SLSI_API pthread_exit %d \n", g_monitoring_thread);
-	pthread_detach(g_monitoring_thread);
-	pthread_exit(&g_monitoring_thread);
+//	pthread_detach(g_monitoring_thread); // pkbuild
```

Comment:
Instead of pkbuild, you need to describe why you comment out in detail.

Reply:
it's mistake. I will remove it. and I write the reason why I delete pthread_ at commit message